src/entities/common/closeable-message-entity.ts

```
@@ -28,7 +28,7 @@ export class CloseableMessageObject extends BaseTappableGameObject {
 
   public close(): void {
     if (this.opacity === 0) {
-      return console.warn("CloseableMessageObject is already closed");
+      return console.warn("CloseableMessageEntity is already closed");
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)